iot_config/src/org_service.rs

```diff
@@ -168,14 +198,28 @@ impl iot_config::Org for OrgService {
 
         tracing::debug!("create helium org request: {request:?}");
 
-        let requested_addrs = request.devaddrs;
-        let devaddr_constraint = org::next_helium_devaddr(&self.pool)
+        let net_id = request.net_id();
+        let requested_addrs = if request.devaddrs >= 8 {
```

Comment:
I'm not sure what the correct answer is here, but if there's a way to consolidate more checks here...
If they pass less than 8, we tell them "insufficient", which I first read as "not enough available".

If they pass 9, this check is good, but you can't checkout an uneven number of devaddrs. And that error message get's returned as "helium addresses unavailable".

Reply:
sure thing; fixed